### DISCUSSION TITLE
MagSafe indicator light synchronization patterns

### DISCUSSION THREAD

**Discussion by lellomascetti** [mhaeuser/Battery-Toolkit#53] [Ideas]
Hi.
After using it for a while at default settings of 80%, I personally felt a bit "uneasy" about the MagSafe indicator being constantly red when on hold, as it would normally mean charging.
Would it be more natural if the indicator was green when the desired charge level was achieved, whenever 100% full or 80% or anything else, and was just default red (not blinking) when charging?
Or maybe an option in settings to choose from various light sync patterns?
As I keep the MagSafe plugged almost all the time, I feel the green color would be somewhat more "relaxing" to always have in sight, and you could also tell from the color if the desired level of charge has been achieved.

**Comment by ghost** [comment]
I don't think he wants to add more complexity to the program.

**Comment by mhaeuser** [comment]
This is basically a duplicate of https://github.com/mhaeuser/Battery-Toolkit/issues/45

**Reply by lellomascetti** [reply]
Sorry, my bad I only searched this topic in discussions, as I don't consider it an issue. Did not see the other post.
Anyway glad to know I'm not the only one sharing these thoughts.
